main.js

```diff
@@ -150,5 +151,5 @@ module.exports = function(options) {
 	return app;
 };
 
-module.exports.Router = express.Router
-
+module.exports.Router = express.Router;
+module.exports.services = serviceMatchers;
```

Comment:
we were talking about exposing the info so radiator can automatically build graphs for each service we're collecting for.

What else would go in next-services?